### ISSUE TITLE
Sending annotated papers by e-mail via SMTP with STARTTLS fails

### ISSUE THREAD

**Issue by mimaoffice** [maelvls/homebrew-amc#56]
I needed to install the following 2 Perl Modules:

    IO::Socket::SSL
    Authen::SASL

with `cpan` as follow:

    /usr/local/bin/cpan IO::Socket::SSL
    /usr/local/bin/cpan Authen::SASL

Note: I had to use the full path to avoid a conflict with MacPorts.

Is is possible to include this in the installation of AMC?

**Comment by maelvls** [comment]
Hi! I'll try to add them today.

**Comment by maelvls** [comment]
I added the two modules, can you try upgrading? I also re-enabled the bottles, installation should be faster now.

**Comment by mimaoffice** [comment]
Now I cannot start AMC anymore. I get the following error message:

```
[michele@fibonacci:~] $ auto-multiple-choice 
Typelib file for namespace 'Gtk', version '3.0' not found at /usr/local/Cellar/auto-multiple-choice/1.4.0_8/libexec/lib/perl5/darwin-thread-multi-2level/Glib/Object/Introspection.pm line 108.
BEGIN failed--compilation aborted at /usr/local/Cellar/auto-multiple-choice/1.4.0_8/lib/AMC/perl/AMC-gui.pl line 23.
```

PS: I deleted my Homebrew installation and installed it again.

**Comment by mimaoffice** [comment]
Changing back the line 44 of the file `auto-multiple-choice.rb` from

    depends_on "gtk+"

to

    depends_on "gtk+3"

solves the problem...

**Comment by maelvls** [comment]
Arghh that’s my fault, it seems like I forgot to switch back to gtk+3 after
a test I did… Sorry 😞

I’ll revert back now

**Comment by mimaoffice** [comment]
> I’ll revert back now

What do you mean with "now"? 😜
Merci!!

**Comment by maelvls** [comment]
Oops I immediately forgot right after sending this message 😅 

I just fixed it in e798c52f25b695983e661617debc3a971890b17a, it should work as before 👍

**Comment by mimaoffice** [comment]
Everything ist fine now. Thank you!
